core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -489,9 +463,7 @@ impl SchedulerCountMetrics {
                 self.num_dropped_on_age_and_status,
                 i64
             ),
-            ("num_dropped_on_capacity", self.num_dropped_on_capacity, i64),
-            ("min_prioritization_fees", min_prioritization_fees, i64),
```

Comment:
wait why separate these from the other metrics? I thought they were fine where they were, even if they aren't strictly a "count"

Reply:
Will revert it then.

Reply:
Fixed.

Reply:
Sorry, I think we're still not on the same page.

Just add `min_prioritization_fee`, `max_prioritization_fee` as fields in the `SchedulerCountMetrics` struct.
Update those fields when we query the min/max from container, and reset them when we report.

Reply:
Updated as you wished, could you check again please.